README.md

```diff
@@ -307,6 +309,25 @@ Options:
 If your job uses variables ([See above](#using-variables-in-the-job-definition)), the variables that should
 be included in the csv data should be specified with the `--variables` option.
 
+### Run VMAF only
+
+This command allows using a configured pipeline to run VMAF measurements on any files without an related autovmaf job.
```

Comment:
Tiny spelling error should be "a related autovmaf job"

Reply:
fixed.